src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+
+    @Override
+    public List<Machine> get() {
```

Comment:
please see the task description

In MachineProducer implementations your method get() should return the list of specific machines. For example:

`List<Bulldozer> get();`
or

`List<Truck> get();`
or

`List<Excavator> get();`